### DISCUSSION TITLE
setup action fails

### DISCUSSION THREAD

**Discussion by rachellanman** [upptime/upptime#737] [General] [answered]
First of all - this is a wonderful repo, thank you for your continual maintenance. 

Regarding my issue: 
I recently setup upptime in my org repository, and I am unable to get the setup action to run. 

The error is the following:
```
Run upptime/uptime-monitor@v1.29.0
Removed legacy .github/workflows
Added new .github/workflows
Removed template files
ERROR TypeError: config.sites.map is not a function
    at Object.updateTemplate (/home/runner/work/_actions/upptime/uptime-monitor/v1.29.0/dist/update-template.js:4[7](https://github.com/usds/uptime-monitoring/actions/runs/4238362879/jobs/7365351376#step:3:8):1)
Error: config.sites.map is not a function
``` 

The run: https://github.com/usds/uptime-monitoring/actions/runs/4238362879/jobs/7365351376

**Comment by AnandChowdhary** [comment, marked as answer]
Your YML syntax is incorrect because "sites" has another "sites" nested inside it: https://github.com/usds/uptime-monitoring/blob/652ea1c610855ed2eafef7916c0df6654a9d2ab1/.upptimerc.yml#L6. Fixing that will fix this issue! :)

**Reply by rachellanman** [reply]
Oh that makes sense. Thanks for the second eye 👀
